public/js/selfoss-events-navigation.js

```diff
@@ -71,7 +71,7 @@ selfoss.events.navigation = function() {
         $(this).addClass('active');
         
         if($(this).hasClass('nav-tags-all')==false) {
-            location.hash = '#' + selfoss.filter.type + '/tag-' + $(this).find('span').html();
+            location.hash = '#' + selfoss.filter.type + '/tag-' + $(this).find('span').html().replace('%', '%25');
```

Comment:
I am afraid that is Chromium issue, Firefox displays the hash decoded. It would be nice if someone with access to Safari could test how it behaves there.